### PR TITLE
Building with Responsive Web design

### PR DESCRIPTION
**Pull request by somoye123**
HTML
- Use of semantic HTML tags to structure the entire webpage
CSS
-  Add corresponding style rules for the HTML 
- Use of media queries for responsive design on different screen sizes

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,903 @@
...
+
+.category,
+.title a {
+  color: #fff;
+  text-align: left;
+}
+
+.title a {
+  margin-top: 0.2em 0;
+}
```
Comment (trekab) [inline main.css]: This property shouldn't have 2 values.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,903 @@
...
+
+.discount {
+  display: none;
+}
+
+.price {
+  margin-top: 0.5em;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
```
Comment (trekab) [inline main.css]: Font family names containing whitespace should be quoted. e.g "Helvetica Neue".

File: main.css [hunk 3]
```diff
@@ -0,0 +1,903 @@
...
+.new-price {
+  font-size: 14px;
+  color: #555;
+}
+
+footer {
+  margin-top: 3em;
+  text-align: center;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
```
Comment (trekab) [inline main.css]: Font family names containing whitespace should be quoted. e.g "Helvetica Neue".

File: main.css [hunk 4]
```diff
@@ -0,0 +1,903 @@
...
+    display: block;
+    width: 80px;
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+
+  .discount h5 {
+    font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+    font-size: 14px;
```
Comment (trekab) [inline main.css]: Font family names containing whitespace should be quoted. e.g "Helvetica Neue".

File: main.css [hunk 5]
```diff
@@ -0,0 +1,903 @@
...
+    margin: 0 10px;
+  }
+
+  .conference h6 span:first-child {
+    font-size: 12px;
+  }
+
+  .timer {
+    font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+    font-size: 14px;
```
Comment (trekab) [inline main.css]: Font family names containing whitespace should be quoted. e.g "Helvetica Neue".

**Review by trekab — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
Hi @somoye123 and @OlawaleJoseph! You have done a nice job on this project so far! There are a few issues to fix to complete, however. These include:
- [x] __Top 3 articles section:__ 
  - For large screens, the text should be positioned at the bottom part of these sections. Your clone appears to have the text positioned mostly near the top part.
  - For small screens, they have a very short height. Make them a bit taller so that they appear more as they do on the original site.
- [x] In the comments below, I have highlighted the `font-family` rule which you have repeated several times. It is better you refactor this out into a class of its own in order to avoid such repetition. 
 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- [x] The brief description for your pull request could use some improvement. Something to keep in mind when writing a good Pull Request summary is: `Your commits tell a story of how you added changes to your project and the pull request summary is the ultimate summary describing briefly what is achieved by all your commits`. Try to keep this in mind on your next pull requests.
- [x] You have done a nice job with the mobile-first approach. :+1: I have noticed that you have added extra(2) break-points though. This is unnecessary. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/)).
- [x] __[OPTIONAL]__ Your use of the `main` tag could use some improvement. The [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) should be used for the dominant content of the `<body>` of a document. In your case, it is being used to achieve something that could be achieved using a `section` tag.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required  🏗️ 🔨 
****
### Hello @somoye123 and @OlawaleJoseph 👋   👋 
You are doing a great job with this project! 👍 You are almost done! 💪 Great job applying the feedback you have received from the previous TSE. 🎖️ 🌟 There are just a few small issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] The previous TSE requested that you position the text on the bottom in the top 3 articles section. Great job fixing this, however, they now appear to be pushed too far to the bottom. In some cases, they are spilling off of the image. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-20 at 12 12 18 AM
The original page 👇 
Screen Shot 2020-06-20 at 12 13 01 AM

- [ ] The elements in your navigation bar on the medium screen size are not aligned properly. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-20 at 12 16 53 AM
The original page 👇 
Screen Shot 2020-06-20 at 12 17 21 AM

- [ ] The footer should remain the same in the small screen size. There is no change in the footer in any of the sizes. Please compare your clone with the original page and fix this.
Your clone👇 
Screen Shot 2020-06-20 at 12 18 55 AM
The original page 👇 
Screen Shot 2020-06-20 at 12 19 21 AM







****
### Kindly make the required changes and resubmit for a new code review once you are ready 🙏


Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by somoye123** [comment]
Screenshot (859)
WhatsApp Image 2020-06-20 at 06 01 30

The first above image is "The Next Web" opened in developer tools in chrome while the next image is "The Next Web" opened in my actual mobile phone

As such am not sure what you mean by "There is no change in the footer in any of the sizes." @BrittanyBlake 

Thanks

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✔

## Congratulations team 🎉. Impressive project 🙌👏👌.

### Onto the next project 🚀.

Happy coding 👨‍💻

**Pull request merged**